README.md

```diff
@@ -26,9 +26,6 @@ From [Kubernetes doc](https://kubernetes.io/docs/tasks/administer-cluster/encryp
 ### 🎁 aks-engine ###
 We have added this feature to aks-engine so that you do not have to worry about any of the manual steps to set this up. Follow this [doc](https://github.com/Azure/aks-engine/blob/master/docs/topics/features.md#azure-key-vault-data-encryption) and this [api model json](https://github.com/Azure/aks-engine/blob/master/examples/kubernetes-config/kubernetes-keyvault-encryption.json) to create your own Kubernetes cluster with Azure Key Vault data encryption. Once the cluster is created, you will see an Azure Key Vault in the same resource group as your cluster, and a new key in the Azure Key Vault.
 
-### Azure Kubernetes Service (AKS) ###
-AKS does encrypt secrets-at-rest, but it doesn’t allow users to bring their own keys.  Keys are managed by AKS.
```

Comment:
Thanks for the PR @TomGeske! 🎉 

We added this snippet because we got alot of questions regarding AKS support for this feature. Instead of removing this whole section, can you pls update it to reflect current state of the project in relations to AKS?

Reply:
agree. adjusted AKS docs and added a link.